universal-application-tool-0.0.1/app/services/question/types/QuestionDefinition.java

```diff
@@ -244,4 +249,13 @@ public boolean equalsIgnoreId(Object other) {
     }
     return false;
   }
+
+  private boolean questionTextAndHelpTextContainsRepeatedEntityNameFormatString() {
+    boolean textMissingFormatString =
+        questionText.translations().values().stream().anyMatch(text -> !text.contains("$this"));
+    boolean helpTextMissingFormatString =
+        questionHelpText.translations().values().stream()
+            .anyMatch(helpText -> !helpText.contains("$this"));
+    return !textMissingFormatString && !helpTextMissingFormatString;
+  }
```

Comment:
Issue (blocking): this will return an error even if the help text is missing altogether, but the error message says it's ok for the help text to be blank

Reply:
I'm not seeing this when I run it locally. 

Error when help text is added and does not include $this:
![image](https://user-images.githubusercontent.com/12072571/118538449-0dd85b80-b703-11eb-85fd-95fb223a2d61.png)

When I get rid of the help text, the question gets created.

Reply:
there's also a unit test that checks when the help text is not present that no error is returned.